core/docker/container-test.sh

```diff
@@ -13,18 +15,21 @@ function test_trino_starts {
 
     local CONTAINER_NAME=$1
     local PLATFORM=$2
-    CONTAINER_ID=$(docker run -d --rm --platform ${PLATFORM} "${CONTAINER_NAME}")
+    CONTAINER_ID=$(docker run -d --rm --platform "${PLATFORM}" "${CONTAINER_NAME}")
 
     set +e
     I=0
-    until RESULT=$(docker exec "${CONTAINER_ID}" trino --execute "SELECT 'success'" 2>/dev/null)
+    until docker inspect "${CONTAINER_ID}" --format "{{json .State.Health.Status }}" | grep -qm 1 '"healthy"'
```

Comment:
The `-m 1` option seems unnecessary, as the GNU grep documentation says for `-q`

> Exit immediately with zero status if any match is found

Reply:
Thanks, removed!